CONTRIBUTING.md

```diff
@@ -102,6 +102,9 @@ If you are adding new APIs, then you might **additionally need to update** [Libr
 
 This helps the AndroidX project keep track of API changes and avoid inadvertently adding APIs or introduce backwards incompatible changes.
 
+**Note:** In case you make a valid violation of Lint, you can use `@Suppress("Rule")` in Kotlin, or `@SuppressLint("Rule")` in Java to suppress the rule. This is now the recommended approach, as opposed to updating the lint baseline (i.e. via `./gradlew updateApiLintBaseline`).
```

Comment:
```suggestion
**Note:** In case you make a valid violation of Lint, you can use `@Suppress("Rule")` in Kotlin, or `@SuppressLint("Rule")` in Java to suppress the rule.
```

Would prefer not to mention the task at all